### PR TITLE
Added intro pitch

### PR DESCRIPTION
**Pull request by Venkatesh-tech**
## Related Issue 

- Added Introduction pitch 

Closes: #70

### Describe the changes you've made

I have added few important lines in the documentation like
- Explaining about the platform
- How this platform works Agenda
- How it creates an interactive section in the platform

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

As far as I have made changes to the Documentation. I checked all the typo errors, grammatical mistakes, and added many meaningful sentences 

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,17 @@
...
+The plight of small and medium companies has gravely wounded the world with serious consequences impacting all the onboard new interns, developers, full-time roles, and individuals during their recruitment phase. To solve these issues and to show a brilliant path to all the individuals listed we proudly present **DevTernShip**, a platform for Internship OnBoarding, projects recruiting, and many more... for students, interns, developers who are looking actively for Internship roles, full-time roles at the moment.
 
-We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
+### Agenda:
+We are on the mission of resolving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth. This platform is the best gateway to show recruiters what interns are capable of.
```
[the remark this answers is not in the document]
Comment (Venkatesh-tech) [reply README.md:16]: Sir, you mean one line space after Heading. Please elaborate

[the remark this answers is not in the document]
Comment (praveenscience) [reply README.md:18]: This is correct.

**Review by praveenscience — APPROVED** [review]
👍

**Review by praveenscience — APPROVED** [review]
LGTM 👍

**Comment by Venkatesh-tech** [comment]
Done

**Review by praveenscience — APPROVED** [review]
👍

**Comment by Venkatesh-tech** [comment]
Thank you. This is my First PR to Open source and I like to contribute more.

**Comment by praveenscience** [comment]
@Venkatesh-tech Bro, how was your feeling about first one. I had the worst, when the merge conflicts occurred! 👎

**Comment by Venkatesh-tech** [comment]
Yeah, it was a tough one. But you shown me a way out Thank you so much, brother, I cannot express it in my words

**Pull request merged**